app/src/components/map/olControls.css

```diff
@@ -0,0 +1,57 @@
+.ol-control {
```

Comment:
I think it is fine having these in a dedicated CSS file, but maybe there is a way to use $vuetify instance helpers to retrieve useful information such as dark/light mode, primary color / background color vars, display breakpoints etc.

There could be some cases where these classes need to be added/removed dynamically and it might be easier if they are in the same SFC, but it is not strictly necessary I guess.